internal/action/command.go

```diff
@@ -207,6 +207,7 @@ func (h *BufPane) TabMoveCmd(args []string) {
 	Tabs.List[idxTo] = activeTab
 	Tabs.UpdateNames()
 	Tabs.SetActive(idxTo)
+	Tabs.Resize()
```

Comment:
nit: move it before `Tabs.UpdateNames()`, to keep it consistent with the code in `AddTab()` and `RemoveTab()`?